functions/script.js

```diff
@@ -0,0 +1,36 @@
+console.log(plus(1, 1));
+console.log(plus(1, "1"));
+console.log(plus(1, +"1"));
+console.log(plus(1, false));
+console.log(plus(1, true));
+console.log(div(10, 3));
+console.log(plus());//undefined->NaN
+console.log(plus(1));//undefined->NaN
+console.log(plus(null, null));//null->0
+console.log(plus("test ", "me"));
+console.log(plus("test ", "me", "again"));
+
+function plus(a, b) {
+	var result = a + b;
+	return result;
+}
+
+function minus(a, b) {
+	var result = a - b;
```

Comment:
actually you could do `return a - b;`.  less code + same sense
the same for another functions.